StrangeIoC/scripts/strange/extensions/context/impl/CrossContext.cs

```diff
@@ -120,7 +121,7 @@ override public IContext AddContext(IContext context)
 		virtual public void AssignCrossContext(ICrossContextCapable childContext)
 		{
 			childContext.crossContextDispatcher = crossContextDispatcher;
-			childContext.injectionBinder.CrossContextBinder = injectionBinder.CrossContextBinder;
+			childContext.injectionBinder = new CrossContextInjectionBinder(injectionBinder.CrossContextBinder);
```

Comment:
Hmm, yeah, I think I was going off of the assumption that this would always be a lazy load instantiation of the InjectionBinder. Is there a time when AssignCrossContext (or AddContext) would be called outside of the childContext's constructor?